### PR TITLE
Client mode & lower log level

### PR DESCRIPTION
**Pull request by vikinatora**

### CODE REVIEW

File: src/main/java/org/peergos/protocol/dht/KademliaEngine.java [hunk 1]
```diff
@@ -46,7 +47,7 @@ public void addIncomingConnection(PeerId peer, Multiaddr addr) {
     public List<PeerAddresses> getKClosestPeers(byte[] key) {
         int k = 20;
         List<Node> nodes = router.find(Id.create(Hash.sha256(key), 256), k, false);
-        System.out.println("Nodes: " + nodes.size());
+        LOG.fine("Nodes: " + nodes.size());
```
Comment (bokoto000) [inline src/main/java/org/peergos/protocol/dht/KademliaEngine.java:50]: This log can be removed or maybe changed to "X Nodes discovered by router". What do you think?

**Review by bokoto000 — APPROVED** [review]

File: src/main/java/org/peergos/protocol/dht/KademliaEngine.java (hunk 1 above)
Comment (bokoto000) [reply src/main/java/org/peergos/protocol/dht/KademliaEngine.java:50]: I don't feel like it conveys the right message as it is.

Comment (vikinatora) [reply src/main/java/org/peergos/protocol/dht/KademliaEngine.java:50]: You're right

File: src/main/java/org/peergos/HostBuilder.java [hunk 2]
```diff
@@ -182,9 +185,9 @@ public static Host build(PrivKey privKey,
                 b.getNetwork().listen(listenAddr);
             }
 
-            b.getConnectionHandlers().add(conn -> System.out.println(conn.localAddress() +
-                    " received connection from " + conn.remoteAddress() +
-                    " on transport " + conn.transport()));
+//            b.getConnectionHandlers().add(conn -> System.out.println(conn.localAddress() +
+//                    " received connection from " + conn.remoteAddress() +
+//                    " on transport " + conn.transport()));
```
[the remark this answers is not in the document]
Comment (vikinatora) [reply src/main/java/org/peergos/HostBuilder.java:190]: My mistake

**Pull request merged**